lib/WP_Auth0_Options.php

```diff
@@ -129,92 +134,124 @@ public function get_logout_url() {
 		return add_query_arg( 'action', 'logout', site_url( 'wp-login.php', 'login' ) );
 	}
 
+	/**
+	 * Get as a string-separated string and parse to array
+	 *
+	 * @return array
+	 */
+	public function get_lock_connections() {
+		$connections = $this->get( 'lock_connections' );
+		$connections = empty( $connections ) ? array() : explode( ',', $connections );
+		return array_map( 'trim', $connections );
+	}
+
+	/**
+	 * Add a new connection to the lock_connections setting
+	 *
+	 * @param string $add_conn - connection name to add
+	 */
+	public function add_lock_connection( $add_conn ) {
```

Comment:
I'd rename this parameter to "connection" or "conn" since the method name already describes what you are doing with it.